### DISCUSSION TITLE
mdast vs. Remark?

### DISCUSSION THREAD

**Discussion by sernaferna** [mdx-editor/editor#587] [Q&A]
As I continue my quest to create some plugins for the editor<sup>1</sup> I included `mdast` as a dependency -- but I see in NPM that `mdast` has been renamed to `remark`. So this may be a silly question, but how does this impact things? I notice the `@mdx-editor/editor` project doesn't even include `mdast` as a dependency but does import from `mdast` -- is that because it's just getting it from `lexical` (and should I do the same)? (I also notice that the project _does_ include `@types/mdast`, for which NPM gives no warnings...)

<sup>1</sup> When I have a better handle on how to write a plugin, I'd be happy to volunteer some documentation [for the docs](https://mdxeditor.dev/editor/docs/extending-the-editor), if there is interest in that.

**Comment by petyosi** [comment]
You don't need the `mdast` package at all (Lexical does not use it). It's basically `mdast-util-from-markdown` and `mdast-util-to-markdown`, and the family of extensions that they need.

**Comment by sernaferna** [comment]
Perfect! Thanks @petyosi -- and can confirm, I was able to remove it from my dependencies and everything still works. 👍🏻
